contracts/test/SafeInternationalHarbour.Bench.spec.ts

```diff
@@ -0,0 +1,209 @@
+		};
+		const signature = await signerWallet.signTypedData(
+			{ chainId, verifyingContract: safeAddress },
+			EIP712_SAFE_TX_TYPE,
+			safeTx,
+		);
+		const tx = await harbour.enqueueTransaction(
+			safeAddress,
+			chainId,
+			safeTx.nonce,
+			safeTx.to,
+			safeTx.value,
+			safeTx.data,
+			safeTx.operation,
+			safeTx.safeTxGas,
+			safeTx.baseGas,
+			safeTx.gasPrice,
+			safeTx.gasToken,
+			safeTx.refundReceiver,
+			signature,
+		);
+		const receipt = (await tx.wait()) as TransactionReceipt;
+		await logGas("erc20_transfer_68b", receipt);
+	});
+
+	it("Enqueuing a transaction with large transaction data", async () => {
+		const { deployer, harbour, chainId, safeAddress } = await deployFixture();
+		const signerWallet = ethers.Wallet.createRandom();
+		const data = `0x${"ff".repeat(1024)}`;
+		const safeTx: SafeTransaction = {
+			to: deployer.address,
+			value: 0n,
+			data,
+			operation: 0,
+			safeTxGas: 0n,
+			baseGas: 0n,
+			gasPrice: 0n,
+			gasToken: ethers.ZeroAddress,
+			refundReceiver: ethers.ZeroAddress,
+			nonce: 0n,
+		};
+		const signature = await signerWallet.signTypedData(
+			{ chainId, verifyingContract: safeAddress },
+			EIP712_SAFE_TX_TYPE,
+			safeTx,
+		);
+		const tx = await harbour.enqueueTransaction(
+			safeAddress,
+			chainId,
+			safeTx.nonce,
+			safeTx.to,
+			safeTx.value,
+			safeTx.data,
+			safeTx.operation,
+			safeTx.safeTxGas,
+			safeTx.baseGas,
+			safeTx.gasPrice,
+			safeTx.gasToken,
+			safeTx.refundReceiver,
+			signature,
+		);
+		const receipt = (await tx.wait()) as TransactionReceipt;
+		await logGas("large_tx_data_1024b", receipt);
+	});
+
+	it("Appending a signature to an existing transaction", async () => {
+		const { deployer, harbour, chainId, safeAddress } = await deployFixture();
+		const signerWallet1 = ethers.Wallet.createRandom();
+		const safeTx: SafeTransaction = {
+			to: deployer.address,
+			value: 0n,
+			data: "0x",
+			operation: 0,
+			safeTxGas: 0n,
+			baseGas: 0n,
+			gasPrice: 0n,
+			gasToken: ethers.ZeroAddress,
+			refundReceiver: ethers.ZeroAddress,
+			nonce: 0n,
+		};
+		const signature1 = await signerWallet1.signTypedData(
+			{ chainId, verifyingContract: safeAddress },
+			EIP712_SAFE_TX_TYPE,
+			safeTx,
+		);
+		await harbour.enqueueTransaction(
+			safeAddress,
+			chainId,
+			safeTx.nonce,
+			safeTx.to,
+			safeTx.value,
+			safeTx.data,
+			safeTx.operation,
+			safeTx.safeTxGas,
+			safeTx.baseGas,
+			safeTx.gasPrice,
+			safeTx.gasToken,
+			safeTx.refundReceiver,
+			signature1,
+		);
+		const signerWallet2 = ethers.Wallet.createRandom();
+		const signature2 = await signerWallet2.signTypedData(
+			{ chainId, verifyingContract: safeAddress },
+			EIP712_SAFE_TX_TYPE,
+			safeTx,
+		);
+		const tx = await harbour.enqueueTransaction(
+			safeAddress,
+			chainId,
+			safeTx.nonce,
+			safeTx.to,
+			safeTx.value,
+			safeTx.data,
+			safeTx.operation,
+			safeTx.safeTxGas,
+			safeTx.baseGas,
+			safeTx.gasPrice,
+			safeTx.gasToken,
```

Comment:
Didn't know you were a tab fanboi.

Reply:
I used the default biome.js settings I got when bootstrapping the app. I don't have a strong preference and can change it if needed.

Reply:
I like tabs: it allows everyone to configure indentation as they prefer!